tests/acceptance/lazy-routeable-engine-test.js

```diff
@@ -66,6 +63,40 @@ test('it should pause to load JS and CSS assets on deep link into a lazy Engine'
   andThen(() => verifyInitialBlogRoute(assert, this.loadEvents, this.application));
 });
 
+test('it should pause to load JS and CSS assets on deep link into a lazy Engine after retrying the transition', function(assert) {
+  assert.expect(7);
+
+  define('dummy/routes/application', () => (Ember.Route.extend({
+      actions: {
+        error: nope
```

Comment:
In general, its easier for me to grok the test if this was just `error() { }` (which happens to be ever so slightly less characters anyways 😝 )...